script/setup

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-npm install -g bower grunt-cli && npm install
+npm install -g grunt-cli && npm install
```

Comment:
Is bower still needed for because of the `prepare` script in WET?

Reply:
Oh yeah it is for core.....But we can change core to not use it at all no?